doc/page-features.h

```diff
@@ -52,6 +52,11 @@ lambda function:
     }
 \endcode
 
+\subsection async-sub3 Async in synchronous interface
+Asynchronous flag can be used with synchronous interface which changes behaviour of annotated method to be alwyas asynchronous.
```

Comment:
"@ipc-sync: true" generates a proxy which has exactly the same API as the interface on the server side.
"@ipc-async: true" generates a proxy whose API is like if all methods were marked @async.